src/ch08/p1_count_syllables.py

```diff
@@ -16,7 +16,10 @@
 from src.ch02 import DICTIONARY_FILE_PATH
 from src.ch02.p1_cleanup_dictionary import cleanup_dict
 
-nltk.download('cmudict')
+if not os.path.exists(os.path.expanduser('~/nltk_data/corpora/cmudict/cmudict')):
+    # FIXME: This is nearly impossible to test.
+    #  Patching os affects every use of os in the module.
+    nltk.download('cmudict')
```

Comment:
This works, but is difficult to test. If there is a better way, please open a PR.